# Description: start gnome-bluetooth in the classic session
# Ubuntu: https://launchpad.net/bugs/924098
# Upstream: ubuntu specific, "classic" is not an upstream session type
Index: gnome-bluetooth-3.2.2/applet/bluetooth-applet.desktop.in.in
===================================================================
--- gnome-bluetooth-3.2.2.orig/applet/bluetooth-applet.desktop.in.in	2012-02-08 12:51:34.647132121 +0100
+++ gnome-bluetooth-3.2.2/applet/bluetooth-applet.desktop.in.in	2012-02-08 12:51:34.699132122 +0100
@@ -11,5 +11,5 @@
 X-GNOME-Bugzilla-Product=gnome-bluetooth
 X-GNOME-Bugzilla-Component=applet
 X-GNOME-Bugzilla-Version=@VERSION@
-AutostartCondition=GNOME3 if-session gnome-fallback
+AutostartCondition=GNOME3 unless-session gnome
 NoDisplay=true
